code/modules/admin/view_variables/mass_edit_variables.dm

```diff
@@ -1,44 +1,39 @@
-/client/proc/cmd_mass_modify_object_variables(atom/A, var_name)
-	set category = "Debug"
-	set name = "Mass Edit Variables"
-	set desc="(target) Edit all instances of a target item's variables"
-
-	var/method = 0 //0 means strict type detection while 1 means this type and all subtypes (IE: /obj/item with this set to 1 will set it to ALL items)
-
-	if(tgui_alert(usr, "Are you sure you'd like to mass-modify every instance of the [var_name] variable? This can break everything if you do not know what you are doing.", "Slow down, chief!", list("Yes", "No"), 60 SECONDS) != "Yes")
+/client/proc/cmd_mass_modify_object_variables(datum/target, var_name)
+	if(tgui_alert(src, "Are you sure you'd like to mass-modify every instance of the [var_name] variable? This can break everything if you do not know what you are doing.", "Slow down, chief!", list("Yes", "No"), 60 SECONDS) != "Yes")
 		return
 
 	if(!check_rights(R_VAREDIT))
 		return
 
-	if(A?.type)
-		method = vv_subtype_prompt(A.type)
+	var/method = 0 //0 means strict type detection while 1 means this type and all subtypes (IE: /obj/item with this set to 1 will set it to ALL items)
```

Comment:
This could use some local declarations instead of a comment, or be a boolean if there are only two methods (and perhaps a better var name in that case).